src/modules/Drbd.rb

```diff
@@ -69,7 +69,7 @@ def main
           "use-rle"
         ],
         "startup" => ["wfc-timeout", "degr-wfc-timeout"],
-        "options" => ["cpu-mask", "on-no-data-accessible"],
+        "options" => ["cpu-mask", "on-no-data-accessible", "quorum"],
```

Comment:
If I understand it correctly there is no UI option for "quorum", but the code makes sure it's kept unchanged (not lost). Is that true?

Reply:
Yes. So far no yast UI for the "options" section in drbd resource configuration, don't have any plan to add it in SLE12SP3. 
Add the code here just in case some user mixed the yast/manual change, it can keep the data in configuration without any changes.